libnavigation-metrics/src/test/java/com/mapbox/navigation/metrics/MapboxMetricsReporterTest.kt

```diff
@@ -4,9 +4,9 @@ import android.os.Parcel
 import com.google.gson.Gson
 import com.mapbox.android.telemetry.Event
 import com.mapbox.android.telemetry.MapboxTelemetry
-import com.mapbox.navigation.base.metrics.MetricEvent
-import com.mapbox.navigation.base.metrics.NavigationMetrics
-import com.mapbox.navigation.metrics.internal.utils.extensions.toTelemetryEvent
+import com.mapbox.navigation.metrics.internal.utils.extentions.MetricEvent
```

Comment:
See comment above.

Reply:
Metrics interfaces should not be spread across three libraries